app/src/main/java/com/github/polypoly/app/utils/Padding.kt

```diff
@@ -0,0 +1,11 @@
+package com.github.polypoly.app.utils
+
+import androidx.compose.ui.unit.dp
+
+class Padding {
```

Comment:
Why create a class with nothing in it except a companion object ❓ . Juste create an object directely.

Reply:
Nice catch, thanks!